ui/lib/apps/SearchLogs/components/Log.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { useState } from 'react'
+import { TextWrap, Pre } from '@lib/components'
+
+interface LogProps {
+  log: string
+}
+
+export default function Log({ log }: LogProps) {
+  const [expanded, setExpanded] = useState<boolean>(false)
```

Comment:
`useState(false)` is fine, it can be inferred, but it's not a big deal.

or maybe we can use @umijs/hooks use-toogle hook: https://hooks.umijs.org/hooks/state/use-toggle

if you would like you can have a try.

Reply:
@baurine addressed, PTAL